src/toil/test/provisioners/aws/awsProvisionerTest.py

```diff
@@ -96,8 +96,7 @@ def _test(self, spotInstances=False):
                        '--retryCount=2',
                        '--clusterStats=/home/']
 
-        toilOptions.extend(['--provisioner=aws',
```

Comment:
This test will probably fail now, the default is only applicable to the cluster utilities - CGCloud is still an option for the actual autoscaling